tests/sentry/models/test_project.py

```diff
@@ -311,13 +317,25 @@ def test_global_disabled(self):
     def test_project_enabled(self):
         user = self.create_user()
         UserOption.objects.set_value(user, "subscribe_by_default", "0")
```

Comment:
don't we want to get `subscribe_by_default` too?

Reply:
Yes that's right I'll convert those now too.